components/Card.vue

```diff
@@ -1,13 +1,34 @@
 <template>
-  <v-card :to="`../item/${item.Id}`">
-    <v-img class="cardImage" :src="imageLink(item.Id)" />
-    <v-card-title>
-      <span>{{ item.Name }}</span>
-    </v-card-title>
-    <v-card-subtitle>
-      <span>{{ item.ProductionYear }}</span>
-    </v-card-subtitle>
-  </v-card>
+  <nuxt-link
+    :to="`/item/${item.Id}`"
+    style="text-decoration: none; color: inherit"
```

Comment:
inline styles? wouldn't a class be better?

Reply:
So it's kind of a hack specifically for cards.

It had a blue line on the bottom and that's the only way I found to remove it...

Reply:
Ok, sounds good.